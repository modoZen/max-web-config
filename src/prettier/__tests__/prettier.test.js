const { getPrettierConfig } = require('../prettier');

describe('Prettier test', () => {
  it('Should return prettier config values', () => {
    expect(getPrettierConfig()).toMatchSnapshot();
  });
});
