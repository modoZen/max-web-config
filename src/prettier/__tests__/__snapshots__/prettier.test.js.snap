// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Prettier test Should return prettier config values 1`] = `
{
  "arrowParens": "always",
  "bracketSameLine": false,
  "bracketSpacing": true,
  "endOfLine": "auto",
  "jsxSingleQuote": true,
  "printWidth": 100,
  "proseWrap": "always",
  "quoteProps": "as-needed",
  "semi": true,
  "singleQuote": true,
  "tabWidth": 2,
  "trailingComma": "all",
  "useTabs": false,
}
`;
